crates/proto/src/serialize/txt/trust_anchor.rs

```diff
@@ -150,6 +154,48 @@ pub enum Entry {
     DNSKEY(Record<DNSKEY>),
 }
 
+/// A resource record as it appears in a zone file
+// like `rr::Record` but with optional TTL field
+#[derive(Debug)]
+pub struct Record<R> {
```

Comment:
Bikeshedding: could/should we have a different name for this? `AnchorRecord`?

Reply:
I think this type is (going to be) used so infrequently that I (fore)see little chances of an import collision, plus it's already possible to disambiguate the types by way of the module prefix: `resource::Record` vs `trust_anchor::Record` and one can always do the rename when importing the type `use trust_anchor::Record as AnchorRecord`